internal/helpers/helpers.go

```diff
@@ -80,6 +81,31 @@ func StripNewLines(stripString string) string {
 	return strings.TrimSuffix(stripString, "\n")
 }
 
+// GreaterThanOrEqualVersion return the given or greater than version
+func GreaterThanOrEqualVersion(a string, b string) (string, error) {
```

Comment:
This and the function below don't seem to be used outside this package, so could they be unexported?

Reply:
Possibly, will check if it will impact something else I'm working on.